### DISCUSSION TITLE
dia-log websites

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-planet-lavitrine#4] [Data Sources] [answered]
There are several websites added to Artsdata from the [dia-log](https://culturelaval.ca/dia-log/) project with Culture Laval.

List of graphs http://kg.artsdata.ca/resource/dia-logGraphs

**Question**: should these be loaded individually by website or all together or by asking Artsdata for all Quebec based events?

## Output Pipeline

Realtime [chasse-galerie-ca JSON](http://api.artsdata.ca/query.json?limit=5&frame=lavitrine/events3&sparql=lavitrine/events3&graph=http://kg.artsdata.ca/culture-creates/footlight/chasse-galerie-ca)

**Comment by christianroy** [comment]
As per my discussion with @GenFab28 today, we believe that asking Artsdata for all Quebec based events is the best solution.

However, we can imagine situations where in the future, for different reasons, we may want to exclue certain websites. So a "blacklist" mechanism could be implemented (but is likely not necessary for the first implementation).

**Comment by saumier** [comment]
@christianroy I have opened issue #12 to enable an API to return all events in the province of Quebec. To do this I will assign each event to the closes Wikidata administrative region and use the Wikidata Place tree to lookup all towns/villages in Quebec with Events. This will remove problems of searching on 'string' that may have different spellings of villages/towns or event provinces.  I have been waiting to do this approach for a while and this is the perfect occasion to implement it :-)

Regarding the "blacklist" it is part of the design of Artsdata that a list of graphs to include or exclude can be easily created. So when we get there it will be straight forward to implement.

**Reply by christianroy** [reply]
Super!

@GenFab28 est-ce que je me trompe ou tu m'as récemment demandé une liste des villes du Québec? On avait identifié la commission de toponymie comme source potentielle, mais Greg a une piste alternative ici, si jamais tu penses que ça peut vous être utile...

**Reply by GenFab28** [reply]
@saumier : absolument! Grégory, est-ce que cette liste des villes-villages + régions administrative a aussi une couche arrondissements pour Montréal? Et est-ce possible de me partager cette liste? Nous cherchons pour le site Vitrine et nos enjeux sont les codes postaux et les longitudes/latitudes.

@christianroy : merci

**Reply by saumier** [reply]
@GenFab28  Oui. There are arrondissements for Montreal in Wikidata Places. At a glance they seem pretty good but I can’t vouch for them 100% because I don’t know what to check them against. 

I will send you a link shortly.

**Comment by saumier** [comment]
@GenFab28 @christianroy Here are 2 links showing part of the places tree in Wikidata for Quebec. Let me know what you think of the quality of this data.

- Quebec places tree https://w.wiki/7Z25
- Urban agglomeration of Montreal  https://w.wiki/7Z27

**Reply by GenFab28** [reply]
@saumier : bien reçu! Ça me semble tout à fait viable. mais il me manquerait les villes-villages. As-tu ça aussi? Cela dit, pour intégration, il va me manquer les codes postaux d'une région. Je ne crois pas pouvoir faire exactement ce dont j'ai besoin.

**Reply by saumier** [reply]
Going one level deeper...

**Reply by GenFab28** [reply]
@saumier    One step forward at La Vitrine ;-)

**Comment by saumier** [comment]
@GenFab28 @christianroy Here is the same table going one level deeper to get more data about the villes-villages and smaller settlements. 
https://w.wiki/7Z6H

**Comment by saumier** [comment, marked as answer]
The dia-log websites listed here http://kg.artsdata.ca/resource/dia-logGraphs will be exported as a single dump.
